test/Dialect/Seq/firreg.mlir

```diff
@@ -80,22 +70,46 @@ hw.module @lowering(%clk: i1, %rst: i1, %in: i32) -> (a: i32, b: i32, c: i32, d:
-  // CHECK-NEXT:         sv.verbatim "{{[{][{]0[}][}]}} = {`RANDOM};" {symbols = [#hw.innerNameRef<@lowering::@_RANDOM_3>]}
-  // CHECK-NEXT:         sv.verbatim "{{[{][{]0[}][}]}} = {{[{][{]1[}][}]}};" {symbols = [#hw.innerNameRef<@lowering::@regE>, #hw.innerNameRef<@lowering::@_RANDOM_3>]}
-  // CHECK-NEXT:         sv.verbatim "{{[{][{]0[}][}]}} = {`RANDOM};" {symbols = [#hw.innerNameRef<@lowering::@_RANDOM_4>]}
-  // CHECK-NEXT:         sv.verbatim "{{[{][{]0[}][}]}} = {{[{][{]1[}][}]}};" {symbols = [#hw.innerNameRef<@lowering::@regF>, #hw.innerNameRef<@lowering::@_RANDOM_4>]}
-  // CHECK-NEXT:         sv.verbatim "{{[{][{]0[}][}]}} = {`RANDOM};" {symbols = [#hw.innerNameRef<@lowering::@_RANDOM_5>]}
-  // CHECK-NEXT:         sv.verbatim "{{[{][{]0[}][}]}} = {{[{][{]1[}][}]}};" {symbols = [#hw.innerNameRef<@lowering::@regA>, #hw.innerNameRef<@lowering::@_RANDOM_5>]}
-  // CHECK-NEXT:         sv.verbatim "{{[{][{]0[}][}]}} = {`RANDOM};" {symbols = [#hw.innerNameRef<@lowering::@_RANDOM_6>]}
-  // CHECK-NEXT:         sv.verbatim "{{[{][{]0[}][}]}} = {{[{][{]1[}][}]}};" {symbols = [#hw.innerNameRef<@lowering::@__rNoSym__>, #hw.innerNameRef<@lowering::@_RANDOM_6>]}
+  // CHECK-NEXT:          %_RANDOM_0 = sv.logic  : !hw.inout<i32>
+  // CHECK-NEXT:          %RANDOM = sv.verbatim.expr.se "`RANDOM" : () -> i32 {symbols = []}
+  // CHECK-NEXT:          sv.bpassign %_RANDOM_0, %RANDOM : i32
+  // CHECK-NEXT:          %_RANDOM_1 = sv.logic  : !hw.inout<i32>
+  // CHECK-NEXT:          %RANDOM_0 = sv.verbatim.expr.se "`RANDOM" : () -> i32 {symbols = []}
+  // CHECK-NEXT:          sv.bpassign %_RANDOM_1, %RANDOM_0 : i32
+  // CHECK-NEXT:          %_RANDOM_2 = sv.logic  : !hw.inout<i32>
+  // CHECK-NEXT:          %RANDOM_1 = sv.verbatim.expr.se "`RANDOM" : () -> i32 {symbols = []}
+  // CHECK-NEXT:          sv.bpassign %_RANDOM_2, %RANDOM_1 : i32
+  // CHECK-NEXT:          %_RANDOM_3 = sv.logic  : !hw.inout<i32>
+  // CHECK-NEXT:          %RANDOM_2 = sv.verbatim.expr.se "`RANDOM" : () -> i32 {symbols = []}
+  // CHECK-NEXT:          sv.bpassign %_RANDOM_3, %RANDOM_2 : i32
+  // CHECK-NEXT:          %_RANDOM_4 = sv.logic  : !hw.inout<i32>
+  // CHECK-NEXT:          %RANDOM_3 = sv.verbatim.expr.se "`RANDOM" : () -> i32 {symbols = []}
+  // CHECK-NEXT:          sv.bpassign %_RANDOM_4, %RANDOM_3 : i32
+  // CHECK-NEXT:          %_RANDOM_5 = sv.logic  : !hw.inout<i32>
+  // CHECK-NEXT:          %RANDOM_4 = sv.verbatim.expr.se "`RANDOM" : () -> i32 {symbols = []}
+  // CHECK-NEXT:          sv.bpassign %_RANDOM_5, %RANDOM_4 : i32
+  // CHECK-NEXT:          %_RANDOM_6 = sv.logic  : !hw.inout<i32>
+  // CHECK-NEXT:          %RANDOM_5 = sv.verbatim.expr.se "`RANDOM" : () -> i32 {symbols = []}
+  // CHECK-NEXT:          sv.bpassign %_RANDOM_6, %RANDOM_5 : i32
+  // CHECK-NEXT:          %_RANDOM_7 = sv.logic  : !hw.inout<i32>
```

Comment:
We now create 7 RANDOM's, apparently because this now rand-initializes "rAnamed" which AFAICT was not being done before.

I didn't expect the set of registers being randomly initialized to change, what's that about? (is this fixing a bug or just fallout of the new code and it's not important)?